libs/blocks/text/text.css

```diff
@@ -12,9 +14,7 @@
 
 .text-block [class^="body-"]:only-child { margin: 0; }
 
-.text-block [class^="heading"] { margin: 0 0 var(--spacing-xs) 0; }
-
-.text-block [class^="detail"] { margin: 0 0 var(--spacing-xxs) 0; }
+.text-block :is([class^="heading"], [class^="detail"]) { margin: 0 0 var(--spacing-xs) 0; }
```

Comment:
This is going to change all the `-detail` default margin. Is this right?

Reply:
@ryanmparrish Yep, according to Consonant that is what they desire.